include/ftl/assert.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <iostream>
+
+#define FTL_STR(s) #s
+#ifndef NDEBUG
+#	define FTL_ASSERT(msg, expr)                                                                                      \
+		if (!(expr)) {                                                                                                 \
+			std::cerr << "FTL Assertion Failure: " << (msg) << ". Expr: " FTL_STR(expr) "\n";                          \
```

Comment:
Can we use fprintf to stderr instead of iostreams? To match the rest of the printing code

Reply:
Sure.